cli/sawtooth_cli/version.py

```diff
@@ -0,0 +1 @@
+VERSION = """0.8.6.dev1"""
```

Comment:
This should maybe be """UNKNOWN""".  But a bigger problem here is that because this file will be overwritten by setup.py, it will cause the git repository to be dirty -- which causes different output from 'git describe', which is used by the get_version script.